overview.md

```diff
@@ -1,5 +1,7 @@
 ## Overview ##
 
+The Synopsys Software Integrity Group is now Black Duck®. If you are an existing Detect user, please manually re-install Black Duck Detect in order to receive future updates. If you are a new user, you may proceed with installing Black Duck Detect.
+
```

Comment:
Please see MR description. Ideally following in the footsteps of Code Sight would be best, hoping you can help with the layout @sig-cpotts